test/helpers/utils.ts

```diff
@@ -0,0 +1,14 @@
+import test from 'tape';
+
+/**
+ * This is a workaround the fact that web3.js doesn't close it's socket connection and provides no way to do so.
+ * Therefore the process hangs for a considerable time after the tests finish, increasing the feedback loop.
+ *
+ * This fixes this by exiting the process as soon as all tests are finished.
+ */
+export function killStuckProcess() {
+  // Don't do this in CI since we need to ensure we get a non-zero exit code if tests fail
+  if (process.env.CI == null) {
+    test.onFinish(() => process.exit(0));
+  }
+}
```

Comment:
I noticed that when I was [playing with Jest a few weeks ago](https://github.com/metaplex-foundation/js-next/blob/loris/jest/tests/helpers/setup.ts#L15-L17). I ended up having a `teardownMetaplex(mx)` method that I would use at the end of my tests. This would close the `_rpcWebSocket` connection like so:

```ts
(mx.connection as any)._rpcWebSocket.close();
```

I wasn't super happy with this solution because we had to access an internal property and we had to teardown each `metaplex` instance. However, it did "properly" kill stuck processes rather than exiting and having to discard CI.

Is there something better we can do? Maybe a hybrid solution that keeps track of all created `metaplex` instance and close them using `onFinish`? What do you think?